src/components/SUE/SueStatus.tsx

```diff
@@ -28,8 +31,22 @@ export const SueStatus = ({
     statusTextColors = {}
   } = sueStatus;
 
-  const backgroundColor = disabled ? statusViewColors?.disabled : statusViewColors?.[status];
-  const textColor = disabled ? statusTextColors?.disabled : statusTextColors?.[status];
+  const backgroundColor =
+    isFilter && disabled
```

Comment:
`disabled` is enough i guess? if it is disabled, it should be grayed out, no matter if is filter or not?

Reply:
ah i see, the problem is that the colors are different?
so we would need more configs for colors for `isFilter`. no it is weird, that we have some colors configured and some not.
please create `statusViewColorsFilter` and `statusTextColorsFilter`, right?

Reply:
I hadn't thought of that since we don't provide customisation for the filter through the server, but it makes sense. I'm adding the following colour codes to the server.
```
      "statusViewColorsFilter": {
        "enabled": "#0f46181a",
        "disabled": "#FFFFFF"
      },
      "statusTextColorsFilter": {
        "enabled": "#141414",
        "disabled": "#A2A2A2"
      },
```

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1033/commits/f5876c73b93ec7c25fc0a3aae93d663812b4a53f